[AMD][gfx1100] test_decompose_mem_bound_mm.py tolerance increase

k-artem opened the pull request:
test_decompose_mem_bound_mm.py tolerance increase for navi3x(gfx11x)

(cherry picked from commit 03c7da05f61890bbf5ae41e23c8df6d5f6805bac) from 

Fixes for CI HUD for gfx1100

Signed-off-by: Artem Kuzmitckii <artem.kuzmitckii@amd.com>

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

iupaikov-amd committed:
[AMD][gfx1100] test_decompose_mem_bound_mm.py tolerance increase for navi3x(gfx11x)

(cherry picked from commit 03c7da05f61890bbf5ae41e23c8df6d5f6805bac)
Signed-off-by: Artem Kuzmitckii <artem.kuzmitckii@amd.com>

dnikolaev-amd committed:
import 'Dict' to fix common_utils.py (#2181)

Bug introduced by
https://github.com/ROCm/pytorch/commit/03c7da05f61890bbf5ae41e23c8df6d5f6805bac

(cherry picked from commit bbd0112a2cfbefbf687f5e907766aba4d030258e)
Signed-off-by: Artem Kuzmitckii <artem.kuzmitckii@amd.com>

k-artem commented:
@pytorchbot label "topic: not user facing"

k-artem commented:
@pytorchbot label "ciflow/rocm"

jeffdaily commented on test/inductor/test_decompose_mem_bound_mm.py:
```diff
@@ -71,31 +73,46 @@ def forward(
+        if rtol is None:
+            rtol = self.rtol
+        if atol is None:
+            atol = self.rtol
```
How does this work?  rtol and atol local to the function.  Modifying them here should not affect call sites like line 88 below.  Right?

k-artem committed:
Fix bug in setup_tolerance

Bug introduced by
https://github.com/ROCm/pytorch/commit/03c7da05f61890bbf5ae41e23c8df6d5f6805bac

Signed-off-by: Artem Kuzmitckii <artem.kuzmitckii@amd.com>

jithunnair-amd commented:
\easycla

molssongroup commented:
/easycla

jeffdaily commented:
@pytorchbot rebase

jeffdaily committed:
lint

jeffdaily committed:
lint

jeffdaily commented:
@pytorchbot merge -f "navi3 test changes only, lint is passing.  too many failures still to make sense of the signal"
